### ISSUE TITLE
New Tables should populate at top of Tables list for easier navigation 

### ISSUE THREAD

**Issue by tdoshin** [microsoft/azuredatastudio#19513]
- Azure Data Studio Version: 1.36.2
- OS Version: Windows 10

Steps to Reproduce:

1. Create new table on Azure Data Studio 
2. Hit "Publish" icon 
3. Select "Update Database
4. Right-click on Tables folder and select "Refresh"

Expected Result:

New table should be seen at top of Tables list drop-down 

Actual Result:

This is not the case; user has to traverse down drop-down to find newly created table. 

For example, in the image below, "Lists" is the name of the newly created table. It should be at the top of the list, but rather, it is 4th on the list which is not very intuitive. This makes navigating objects with lots of tables a bit strenuous. 

![image](https://user-images.githubusercontent.com/72947001/170142761-827f5972-f6f7-424d-8d3d-2cc66a2495cd.png)

FYI @alanrenmsft @caohai 

Does this issue occur when all extensions are disabled?: No

**Comment by Charles-Gagnon** [comment]
I don't think this necessarily should be the case - many people would prefer it to be alphabetical like it is currently.

Rather, what the "best" solution would be is to have the ability to choose how the list is sorted. So then the user could choose themselves whether the list is alphabetical, by age, or some other metric.

**Comment by caohai** [comment]
> Expected Result:
> 
> New table should be seen at top of Tables list drop-down

Can you elaborate on why this should be the expected result? Once a new table is published, it becomes an existing table, just like other tables, I'm not sure why it should be on the top.

If you are saying we want a different sorting option for tables in OE, then it's a different story but that has nothing to do with table designer itself.

I also tested in SSDT and the newly created table `[YYYYYTable]` is not at the top.
![image](https://user-images.githubusercontent.com/21186993/170149272-a7ff0253-1921-41e5-94ef-08bda13a0444.png)

**Comment by tdoshin** [comment]
I agree with @Charles-Gagnon  I believe that we should provide users the flexibility to decide how they want their tables to be sorted. We can start with two metrics (sort by "recently added" OR sort by "alphabetical") and continue to add more metrics as needed as we iterate on this feature. @caohai
